WindowsServerDocs/storage/storage-replica/storage-replica-overview.md

```diff
@@ -143,7 +143,9 @@ With its higher than zero RPO, asynchronous replication is less suitable for HA
 
 -   Network bandwidth and latency with fastest storage. There are physical limitations around synchronous replication. Because Storage Replica implements an IO filtering mechanism using logs and requiring network round trips, synchronous replication is likely make application writes slower. By using low latency, high-bandwidth networks as well as high-throughput disk subsystems for the logs, you will minimize performance overhead.  
 
--   The destination volume is not accessible while replicating. When you configure replication, the destination volume dismounts, making it inaccessible to any reads or writes by users. Its driver letter may be visible in typical interfaces like File Explorer, but an application cannot access the volume itself. Block-level replication technologies are incompatible with allowing access to the destination target's mounted file system in a volume; NTFS and ReFS do not support users writing data to the volume while blocks change underneath them.  
+-   The destination volume is not accessible while replicating in Windows Server 2016 RTM. When you configure replication, the destination volume dismounts, making it inaccessible to any reads or writes by users. Its driver letter may be visible in typical interfaces like File Explorer, but an application cannot access the volume itself. Block-level replication technologies are incompatible with allowing access to the destination target's mounted file system in a volume; NTFS and ReFS do not support users writing data to the volume while blocks change underneath them. 
+
+In Windows Server Version 1709 (aka "RS3") the new Test-Failover option was added. This now supports temporarily mounting a read-write snapshot of the destination volume for backups, testing, etc. See https://aka.ms/srfaq for more info.
```

Comment:
Same comment here as in the first PR - can you drop RTM and RS3? Windows Server 2016, Windows Server (Semi-Annual Channel), or Windows Server, version 1709. Thanks!